public/js/selfoss-ui.js

```diff
@@ -176,13 +178,11 @@ selfoss.ui = {
     entrySelect: function(entry) {
         if (selfoss.ui.selectedEntry !== null) {
             selfoss.ui.selectedEntry.removeClass('selected');
-            selfoss.ui.selectedEntry.attr('aria-current', 'false');
```

Comment:
Instead you need to mark the title of selected entry here: `$('.entry-title > .entry-title-link', selfoss.ui.selectedEntry).attr(...)`

Reply:
> Instead you need to mark the title of selected entry here: `$('.entry-title > .entry-title-link', selfoss.ui.selectedEntry).attr(...)`

Done the changes. Please check. However, I'm yet to test whether it works properly or not.